drizzlepac/runastrodriz.py

```diff
@@ -1677,6 +1679,45 @@ def update_active_wcs(filename, wcsname, logfile=None):
     if logfile:
         _updateTrlFile(logfile, update_msg)
 
+def _update_wcs_fit_keywords(fltfile, flcfile):
+    """Update the header of the FLT file with the a posteriori fit results"""
+    fit_kws_sci = [('RMS_RA', -1.0), ('RMS_DEC', -1.0),
+                   ('CRDER1', -1.0), ('CRDER2', -1.0),
+                   ('NMATCHES', 0), ('FITGEOM', 'N/A'),
+                   ('HDRNAME', '')]
+
+    hdulist = fits.open(fltfile, mode='update')
+    hdulist_flc = fits.open(flcfile)  # source header
+
+    if 'HISTORY' in hdulist[0].header:
+        after_kw = None
+        before_kw = 'HISTORY'
+    elif 'ASN_MTYP' in hdulist[0].header:
+        after_kw = 'ASN_MTYP'
+        before_kw = None
+    else:
+        after_kw = hdulist[0].header.cards[-1][0]
+        before_kw = None
+
+    hdulist[0].header.set('UPWCSVER', value=hdulist_flc[0].header['UPWCSVER'],
+                          comment="Version of STWCS used to update the WCS",
+                          after=after_kw, before=before_kw)
+    hdulist[0].header.set('PYWCSVER', value=hdulist_flc[0].header['UPWCSVER'],
```

Comment:
Should the value from "FLC" be "PYWCSVER"?  Also, should the comment be "Version of PYWCS used to update the WCS "?  I am unsure about the commentary portion of the FITS Card, but I thought I should confirm.

Reply:
It looks as if you found a cut-and-paste error I overlooked.  Fixed in d79d1af.